packages/default/scss/grid/_layout.scss

```diff
@@ -376,8 +376,15 @@
             vertical-align: middle;
         }
         .k-button + .k-button {
```

Comment:
Initially, we were supposed to have only buttons in the toolbar, hence the selector. It should now be `* + *`. Of course, since now we can have toolbar and button groups, that should be immediate descendant (child) selector or `>`. The end form of the selector should be `> * + *`

Reply:
Fixed.